tests/base/symmetric_tensor_39.cc

```diff
@@ -0,0 +1,70 @@
+#include <iostream>
+#include <fstream>
+#include <type_traits>
+
+#include <deal.II/base/symmetric_tensor.h>
+#include <deal.II/base/vectorization.h>
+
+using namespace dealii;
+
+int main()
+{
+  std::string logname = "output";
+  std::ofstream logfile(logname);
+
+#if defined DEAL_II_COMPILER_VECTORIZATION_LEVEL
+  std::cout << "Vectorization preprocessor variable found!" << std::endl;
+#endif
+
+#if DEAL_II_COMPILER_VECTORIZATION_LEVEL == 0
+  std::cout << "Value of DEAL_II_COMPILER_VECTORIZATION_LEVEL represents no vectorization" << std::endl;
+#elif DEAL_II_COMPILER_VECTORIZATION_LEVEL == 1
+  std::cout << "Value of DEAL_II_COMPILER_VECTORIZATION_LEVEL represents SSE" << std::endl;
+#elif DEAL_II_COMPILER_VECTORIZATION_LEVEL == 2
+  std::cout << "Value of DEAL_II_COMPILER_VECTORIZATION_LEVEL represents AVX" << std::endl;
+#else
+  std::cout << "Value of DEAL_II_COMPILER_VECTORIZATION_LEVEL not in {0,1,2}" << std::endl;
+#endif
+
+  const unsigned int dim = 3;
+
+  SymmetricTensor<4,dim,VectorizedArray<double>> I;
+  SymmetricTensor<2,dim,VectorizedArray<double>> A;
+  SymmetricTensor<2,dim,VectorizedArray<double>> B;
```

Comment:
`SymmetricTensor<2,dim,VectorizedArray<double> >` in order for this not to require `C++11`.